openghg/store/base/_base.py

```diff
@@ -126,7 +134,7 @@ def assign_data(
 
                 # Make sure all the metadata is lowercase for easier searching later
                 # TODO - do we want to do this or should be just perform lowercase comparisons?
-                meta_copy = to_lowercase(d=meta_copy, skip_keys=skip_keys)
+                # meta_copy = to_lowercase(d=meta_copy, skip_keys=skip_keys)
```

Comment:
Just to check - should this still be commented out after your changes around booleans / strings etc.?